cms/templates/js/certificate-editor.underscore

```diff
@@ -19,6 +19,10 @@
                 <textarea id="certificate-description-<%= uniqueId %>" class="certificate-description-input text input-text" name="certificate-description" placeholder="<%= gettext("Description of the certificate") %>" aria-describedby="certificate-description-<%=uniqueId %>-tip"><%= description %></textarea>
                 <span id="certificate-description-<%= uniqueId %>-tip" class="tip tip-stacked"><%= gettext("Description of the certificate") %></span>
             </div>
+            <div class="field text actual-course-title">
+                <label class="actual-course-title"><b><%= gettext("Course Title") %></b></label>
+                <span class="actual-title"><%= course.get('name') %></span>
+            </div>
```

Comment:
Maybe it's just me, but I'm not seeing any sort of specification here that says 'Show the name on the next line'?  I guess I was expecting some CSS to be included with the markup?